app/views/posts/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<div>
+    <%= form_with model: @newPost, url: user_posts_path(@user) do |f| %>
+      <%= f.label :title %><br>
+      <%= f.text_field :title %><br>
+      <%= f.label :text %><br>
+      <%= f.text_area :text %><br>
+      <%= f.submit %>
+    <% end %>
+</div>
```

Comment:
- [ ] It would be better if users can interact with the application. Once the app starts, we are presented with the screen below
![Screenshot from 2022-09-24 00-50-34](https://user-images.githubusercontent.com/46388113/192073383-b2d01e2b-5bbd-464c-9604-b3eeccbe9832.png)
And then I try to create a post from the link at the top, here we have only 2 links, home and create_new_post. After clicking on the `create new post` button i get the error below
![Screenshot from 2022-09-24 00-50-53](https://user-images.githubusercontent.com/46388113/192073462-0def82a2-41d7-4c18-9543-1e072633451d.png)

Here is the problem, since we have nested routes, ie. post inside a user, we need to create a user before creating a post so the url will look something similar to `users/1/posts/create`. Kindly investigate around this and try and fix it :man_technologist:

Reply:
Hey @addod19, thank you for the suggestion. I understand where you are coming from. I am able to access the home page as below. I would assume you had fed in data into the database because at this point the project requirements don't need us to have the new user interface. Is there something I am missing?

![image](https://user-images.githubusercontent.com/59999191/192108678-f4d7ffe3-9450-47b9-94cd-e0a7d5e0baae.png)

Reply:
> Hey @addod19, thank you for the suggestion. I understand where you are coming from. I am able to access the home page as below. I would assume you had fed in data into the database because at this point the project requirements don't need us to have the new user interface. Is there something I am missing?
> 
> ![image](https://user-images.githubusercontent.com/59999191/192108678-f4d7ffe3-9450-47b9-94cd-e0a7d5e0baae.png)

Yeah sure....My bad, it should have been marked as optional. Can you kindly submit another code review request as you have the screenshot that shows the data :+1:

Reply:
Alright thank you for clarifying.